src/unsup/sub-add.c

```diff
@@ -270,6 +284,12 @@ int main (int argc, char *argv[]) {
     printf ("(%ld equivalent at a +-%ld level) ", NrEquivs, equiv);
   printf ("found out of %ld.\n", N * N2);
 
+  if (return_error) {
```

Comment:
Maybe a dump question, but the sub-add function does not return any errors if the `-e` option is not used, does it? Is this intended?

Reply:
Correct. The original code would display that the files are different but would not return an error.

This patch provides a switch to return an error if the files are different. This makes scripting much easier. This is provided as an option for backward compatibilities.

Reply:
Sounds good. :-)